File: backends/python-fastapi/app/usecase/invitation/dto.py
"""
招待ユースケース Dto モジュール。

Author: Satoshi Nagashiba <satoshi.nagashiba@gmail.com>
"""
from dataclasses import dataclass


@dataclass
class InvitationResultDto:
    """招待結果 Dto。"""

    token: str
    url: str
    display_url: str

File: backends/ts-hono/src/usecase/invitation/dto.ts
/**
 * 招待ユースケース DTO モジュール。
 *
 * @author Satoshi Nagashiba <satoshi.nagashiba@gmail.com>
 */

/** 招待トークン取得出力 DTO。 */
export interface InvitationCurrentOutput {
  token: string;
  url: string;
  displayUrl: string;
}

/** 招待トークン発行出力 DTO。 */
export interface InvitationIssueOutput {
  token: string;
  url: string;
  displayUrl: string;
}
